inst/include/Rcpp/api/meat/Environment.h

```diff
@@ -44,6 +44,15 @@ Environment_Impl<StoragePolicy>::Environment_Impl( int pos ){
    Storage::set__(env) ;
 }
 
+inline Environment new_env(int size = 29) {
+    Function newEnv("new.env");
```

Comment:
Should we be fancy and memoise / keep a once init'ed `Function` ?  Probably not worth it.

Reply:
We could do that (just make it a static object so it's only initialized once on first call). Certainly doesn't hurt but agree we don't gain much.